docs/Documentation/Systems/Vermillion/systemconfig.md

```diff
@@ -6,3 +6,15 @@ grand_parent: Systems
 ---
 
 # System Configuration
+##
+
+---
+### Vermilion Partitions
```

Comment:
Partition chart could easily be created in system overview or elsewhere with more description.